### DISCUSSION TITLE
How will existing MIR certificates be mapped?

### DISCUSSION THREAD

**Discussion by mirceahasegan** [CardanoSolutions/ogmios#340] [Q & A]
Hello,
Since MIR certificates are removed from the cddl and from ogmios, how will they be mapped during chain sync?
Thank you!

**Comment by KtorZ** [comment]
They have been unified under governance actions. More specifically, they correspond to the [`GovernanceAction<TreasuryTransfer>`](https://github.com/CardanoSolutions/ogmios/blob/master/docs/static/cardano.json#L872-L894) in the JSON-schema.

https://github.com/CardanoSolutions/ogmios/blob/2b3370b53384a992a0f81795946d94fc7ce210b9/docs/static/cardano.json#L872-L894
